lhotse/kaldi.py

```diff
@@ -231,8 +225,8 @@ def export_to_kaldi(
         # segments
         save_kaldi_text_mapping(
             data={
-                cut.supervisions[0].id: f"{cut.recording_id} {cut.start} {cut.end}"
-                for cut in cuts
+                sup.id: f"{sup.recording_id} {sup.start} {sup.start + sup.duration}"
```

Comment:
You can also just use `sup.end` (`SupervisionSegment` has an `end()` [property](https://github.com/lhotse-speech/lhotse/blob/b41e4f8c57954b94790f2d05de310a11071512bc/lhotse/supervision.py#L187).